quadrat/child-experimental-theme.json

```diff
@@ -25,29 +25,34 @@
 				"customFontSize": true,
 				"customLineHeight": true,
 				"fontSizes": [
+					{
+						"name": "Tiny",
+						"slug": "tiny",
+						"size": "15px"
+					},
 					{
 						"name": "Small",
-						"size": "17px",
+						"size": "22px",
```

Comment:
These declarations generate presets available by default the editor. This makes me think they should follow the [paragraph sizing presented in the designs here](https://www.figma.com/file/EDoXTvsZt4lMHItzOczFQw/?node-id=443%3A4):

<img width="537" alt="Screen Shot 2021-04-14 at 10 07 42 AM" src="https://user-images.githubusercontent.com/5375500/114724259-375f2b00-9cf0-11eb-87be-7d275aad5426.png">

For the heading sizes, I think we should make those custom variables that we can change based on a media query. Thoughts?

Reply:
cc @beafialho

Reply:
+1 to the custom variables based on a media query for Headings.

Still not sure about the sizes for presets.

Reply:
Correct @jffng @danieldudzic, the final styles are on [this screen](https://www.figma.com/file/EDoXTvsZt4lMHItzOczFQw/Quadrat?node-id=760%3A112) in the file.

Reply:
Added a link to that screen in the main issue